augur/config.py

```diff
@@ -129,125 +157,123 @@
+                    "confidence_interval": 95,
+                    "contamination": 0.1,
+                    "switch": 1,
+                    "workers": 1,
+                    "training_days": 1000,
+                    "anomaly_days": 14
+                },
+                "linux_badge_worker": {
+                    "port": 50400,
+                    "switch": 1,
+                    "workers": 1
+                },
+                "metric_status_worker": {
+                    "port": 50500,
+                    "switch": 0,
+                    "workers": 1
+                },
+                "pull_request_worker": {
+                    "port": 50600,
+                    "switch": 1,
+                    "workers": 1
+                },
+                "repo_info_worker": {
+                    "port": 50700,
+                    "switch": 1,
+                    "workers": 1
+                },
+                "value_worker": {
+                    "port": 50800,
+                    "scc_bin": "scc",
+                    "switch": 0,
+                    "workers": 1
+                },
+                "contributor_worker": {
+                    "port": 50900,
+                    "switch": 0,
+                    "workers": 1
+                },
+                "gitlab_issues_worker": {
+                    "port": 51000,
+                    "switch": 1,
+                    "workers": 1
+                },
+                "release_worker": {
+                    "port": 51100,
+                    "switch": 1,
+                    "workers": 1
+                },
+                "gitlab_merge_request_worker": {
+                    "port": 51200,
+                    "switch": 0,
+                    "workers": 1
+                },
+                "message_insights_worker": {
+                    "port": 51300,
+                    "switch": 0,
+                    "workers": 1,
+                    "insight_days": 30,
+                    "models_dir": "message_models"
+                },
+                "pull_request_analysis_worker": {
+                    "port": 51400,
+                    "switch": 0,
+                    "workers": 1,
+                    "insight_days": 30
+                },
+                "discourse_analysis_worker":{
+        	    "port" : 51500,
+        	    "switch": 0,
+        	    "workers": 1
 
-	    },
-	    "clustering_worker": {
-            "port": 51600,
-            "switch": 0,
-            "workers": 1,
-	    "max_df" : 0.9,
-	    "max_features" : 1000,
-	    "min_df": 0.1,
-	    "num_clusters" : 4
+        	    },
+        	    "clustering_worker": {
+                    "port": 51600,
+                    "switch": 0,
+                    "workers": 1,
+        	    "max_df" : 0.9,
+        	    "max_features" : 1000,
+        	    "min_df": 0.1,
+        	    "num_clusters" : 4
 
-        }
-        },
+                }
+            },  
```

Comment:
This indentation is also incorrect. It needs to follow the formatting of the rest of the object.